pkg/controllers/podgroup/pg_controller_handler.go

```diff
@@ -72,10 +72,30 @@ func (pg *pgcontroller) addReplicaSet(obj interface{}) {
 
 	if *rs.Spec.Replicas == 0 {
 		pgName := batchv1alpha1.PodgroupNamePrefix + string(rs.UID)
+		klog.V(4).Infof("Delete podgroup %s for replicaset %s/%s spec.replicas == 0",
+			pgName, rs.Namespace, rs.Name)
 		err := pg.vcClient.SchedulingV1beta1().PodGroups(rs.Namespace).Delete(context.TODO(), pgName, metav1.DeleteOptions{})
 		if err != nil && !apierrors.IsNotFound(err) {
 			klog.Errorf("Failed to delete PodGroup <%s/%s>: %v", rs.Namespace, pgName, err)
 		}
+	} else if *rs.Spec.Replicas > 0 {
```

Comment:
else is no need and we should add a comment to explain why this is added.

Reply:
Sorry, I don't understand what you mean by else is not needed. Are you saying that the webhook already guarantees that replicas are unsigned, so there's no need to write a > 0 check? 
The comment I'll add later.

Reply:
Sorry for misunderstanding, I mean we can use `if *rs.Spec.Replicas > 0` directly and no need to use `else if`.

Reply:
Modified based on review comments.